src/Stakeholders/components/FocalPeople/ListItem/index.js

```diff
@@ -107,15 +107,23 @@ class FocalPeopleListItem extends Component {
   };
 
   render() {
+    const sideSpan = { xxl: 1, xl: 1, lg: 1, md: 2, sm: 3, xs: 3 };
```

Comment:
I think you should refactor these to be constants outside this component like you did with other components